docs/roles/cp4d_db2wh_backup.md

```diff
@@ -0,0 +1,63 @@
+cp4d_db2wh_backup
+==========
+
+This role runs a backup procedure from a CloudPak for Data DB2 Warehouse instance and stores the backup files in a targetted local folder.
+At the end of the backup process, you will find the required files to run a successful restore process in the chosen `DB2WH_BACKUP_DIR`:
+
+- DB2 backup files i.e `BLUDB.0.db2inst1.DBPART000.202XXXXXXXXXXX.001`
+- DB2 keystore files (.p12 and .sth)
+- DB2 instance master key label file (.kdb)
+
+Role Variables
+--------------
+
+### Required Variables
```

Comment:
Can you update the README to match the new(ish) format we are using now: e.g. https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/cp4d_db2wh/README.md#role-variables

Reply:
Fixed.